src/elm/Components/Logs.elm

```diff
@@ -50,6 +54,8 @@ type alias Props msg =
     }
 
 
+{-| LogLine : alias for an object containing a msg.
```

Comment:
some of these comments i think could be more descriptive in a way that helps a reader understand what the alias is meant for.
in this case i dont think the comment helps much.

```suggestion
{-| LogLine : alias that contains data necessary to render a log line.
```

Reply:
will tackle this and similar comments in another pr.